code/exercises/affine-cipher/example.scm

```diff
@@ -0,0 +1,89 @@
+(import (rnrs))
+
+
+(define (encode key text)
+  ((get-translator key) (string-downcase text) 'encode))
+
+(define (decode key text)
+  ((get-translator key) text 'decode))
```

Comment:
The translator objects are throw away here, but I think building a translator is definitely the way to do this exercise.  We could also extend the tests later to something fancy, like build the generator, and encode then decode the output.